### DISCUSSION TITLE
Is this project still being maintained? (spoiler: yes it is)

### DISCUSSION THREAD

**Discussion by foogazy** [uNetworking/uWebSockets.js#786] [Q&A] [answered]
Hello, I hope you are well. 

I was wondering if this project is still active?  The reason for this question is that, although the project had updates not too long ago, the [smallest pull requests](https://github.com/uNetworking/uWebSockets.js/pull/719) are not being merged and the [builds are failing](https://github.com/uNetworking/uWebSockets.js/actions) for the latest version.

If it is still being maintained, is there anything we can do to help with fixing these issues?

Many thanks.

**Comment by e3dio** [comment]
PR was merged couple days ago, you are saying you want your linked PR merged? You should comment on that PR. I don't use TypeScript. I don't know about the failing job, looks like Mac failed but Linux/Windows completed

**Reply by foogazy** [reply]
Hello! Thank you for your message. The problem is, the last commit was 5th of June and it was a fix on Readme but the thing that concerns me the most is that the main maintainer appears to have deleted their account.

**Reply by e3dio** [reply]
Wrong last commit was Aug 5th when multiple people commented on and approved PR https://github.com/uNetworking/uWebSockets.js/commits/binaries

**Comment by shaharmor** [comment]
There also hasn't been any release for a few months while there are new features that we want to use

**Reply by e3dio** [reply]
Clone and `make` for latest build, takes like 2 seconds, or you prefer new release for every commit ?

**Comment by knight10-ux** [comment]
Actually this is kinda important, it seems that Alex Huntman deleted his github account?

**Reply by foogazy** [reply]
That's true! And actually that's my main concern too.

**Reply by joshxyzhimself** [reply]
1. Our reliable sources say he might again be currently in a CIA blacksite at some defunct oilrig nearby Guantanamo Bay being interrogated for the nth time on where he got the alien technology to create uws.
2. He's okay, given the recent commits and push on another GitHub account on the upstream repos we can confirm he's given his ssh keys printed in comic sans and a tamagotchi with 5g plash speed router to continue his work on this alien technology.

**Reply by uNetworkingAB** [reply]
Lol

**Reply by uNetworkingAB** [reply]
Yes I figured I did not want big tech keeping 10+years record on me so I deleted that old account and made this new one instead.

**Comment by uNetworkingAB** [comment]
After having my nipples connected to a car battery for 2 months I escaped Guantanamo Bay on a sea turtle

**Reply by uNetworkingAB** [reply]
Which in layman's terms means I now have Watching: All activity for this repo. Whoops did not get messages.

**Comment by uNetworkingAB** [comment, marked as answer]
If you look at the main repo, which is where 96% of this project is kept, it's actively developed with tons of new features recently: https://github.com/uNetworking/uWebSockets/releases

This repo, uWebSockets.js, is just a thin wrapper that does nothing other than wrap the main project for Node.js users. Therefore this repo sees very few changes and can stall behind the main repo until significant changes have accumulated and need a release.

Releases are not just a "click and be done"; there are billion-dollar companies using this code in prod so naturally I don't want to create new releases every minute since every release comes with risk of failure. Basically don't fix what's not broken unless you have to.

You can expect a new release within 2 months.

**Comment by Davut-Atajanov** [comment]
I am wondering this on 4th quarter of 2024, planning on using this on big scale production app

**Reply by uNetworkingAB** [reply]
The project is 8 years old. Last release was 1 month ago. Last release in main project was 2 weeks ago. It's not going to have daily changes just for the sake of having daily changes. The project is older than Deno, Bun and countless other "frameworks" that have come and gone. Chances are Node.js itself will be deprecated before this project.
